### PR TITLE
Update Stock Bug Fix to fix Tepidizer target temperature area

### PR DESCRIPTION
**Pull request by FrancoisDupayrat**
Discussed extensively on Discord.

### CODE REVIEW

File: StockBugFix/StockBugsPatches.cs [hunk 1]
```diff
@@ -452,5 +453,61 @@ public static class SolidTransferArm_OnSpawn_Patch {
...
+					CodeInstruction instruction = instructionList[i];
+					if (instruction.opcode == OpCodes.Call && instruction.operand != null &&
+						instruction.operand.Equals(targetMethod)) {
+						targetIndex = i;
+						break;
+					}
+				}
+				if (targetIndex == -1) {
+#if DEBUG
+					PUtil.LogError("Target method GetNonSolidCells not found.");
```
Comment (peterhaneve) [inline StockBugFix/StockBugsPatches.cs:480]: `LogError` will blackhole the game, which is not an ideal thing to do in a `DEBUG`-only code section. Consider downgrading to a `Warning` and removing `#if DEBUG`.

File: StockBugFix/StockBugsPatches.cs [hunk 2]
```diff
@@ -452,5 +453,61 @@ public static class SolidTransferArm_OnSpawn_Patch {
...
+		/// <summary>
+		/// Applied to SpaceHeater to fix Tepidizer target temperature area being too large.
+		/// </summary>
+		[HarmonyPatch(typeof(SpaceHeater), "MonitorHeating")]
+		public static class SpaceHeater_MonitorHeating_Patch {
+			/// <summary>
+			/// Transpiles MonitorHeating to replace the GetNonSolidCells call.
+			/// </summary>
+			internal static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> method) {
+				List<CodeInstruction> instructionList = method.ToList();
```
Comment (peterhaneve) [inline StockBugFix/StockBugsPatches.cs:466]: Nit: If you use the `List<T>.List(IEnumerable<T>)` constructor, then `System.Linq` can be avoided, like in L248

File: StockBugFix/StockBugsPatches.cs (hunk 1 above)
Comment (FrancoisDupayrat) [reply StockBugFix/StockBugsPatches.cs]: Fixed

File: StockBugFix/StockBugsPatches.cs (hunk 2 above)
Comment (FrancoisDupayrat) [reply StockBugFix/StockBugsPatches.cs]: Fixed

**Pull request merged**